src/main/java/org/folio/oaipmh/helpers/records/RecordMetadataManager.java

```diff
@@ -116,9 +128,17 @@ public JsonObject populateMetadataWithItemsData(JsonObject srsInstance, JsonObje
    *
    * @param itemData         - json of single item
    * @param marcRecordFields - fields list to be updated with new one
+   * @param suppressedRecordsProcessing - include suppressed flag in 952 field?
    */
-  private void updateFieldsWithItemEffectiveLocationField(JsonObject itemData, List<Object> marcRecordFields) {
+  private void updateFieldsWithItemEffectiveLocationField(JsonObject itemData,
+                                                          List<Object> marcRecordFields,
+                                                          boolean suppressedRecordsProcessing) {
     Map<String, Object> effectiveLocationSubFields = constructEffectiveLocationSubFieldsMap(itemData);
+    if (suppressedRecordsProcessing) {
+      final Boolean itemSuppressedFromDiscovery = itemData.getBoolean(INVENTORY_SUPPRESS_DISCOVERY_FIELD);
+      int subFieldValue = itemSuppressedFromDiscovery == null || !itemSuppressedFromDiscovery ? 0 : 1;
```

Comment:
Taking into account that we are already used "apache common lang3" this confditions can be replaced by BooleanUtils.isFalse(). How fo you think, does it make sence?

Reply:
Done.